### PR TITLE
Removing parquet.hadoop export

### PR DESCRIPTION
**Pull request by CuriousVini**
Issue: https://issues.cask.co/browse/CDAP-5768
Build: http://builds.cask.co/browse/HYP-DBT9-1

### CODE REVIEW

File: core-plugins/pom.xml [hunk 1]
```diff
@@ -199,7 +199,6 @@
               co.cask.hydrator.plugin.validator.*;
               co.cask.hydrator.plugin.common;
               parquet.avro.*;
```
Comment (albertshau) [inline core-plugins/pom.xml:201]: can this be removed too?

Comment (rohitsinha54) [inline core-plugins/pom.xml:201]: I think the ParquetInputFormat comes from here

Comment (CuriousVini) [inline core-plugins/pom.xml:201]: I think we can't because of parquet.avro.AvroParquetInputFormat.

**Comment by albertshau** [comment]
lgtm assuming writing to and reading from parquet has been tested on both mapreduce and spark

**Comment by albertshau** [comment]
also, shouldn't this fix go to the 1.3 branch?

**Comment by CuriousVini** [comment]
closing and reopening against release branch

**Pull request closed without merging**